src/dictionary.js

```diff
@@ -0,0 +1,24 @@
+const https = require("https");
+const fs = require("fs");
+const generateList = async () => new Promise((resolve, reject) => {
+    let wordList = [];
+    if (fs.existsSync("../dictionary.json")) {
+        wordList = fs.readFileSync("../dictionary.json");
```

Comment:
I'd like to have a local cache dir, maybe homegames/local by default that we store files to, but this is okay for now

Reply:
I agree and moved stuff to a local directory